src/lib.rs

```diff
@@ -29,26 +29,36 @@
 //! This crate uses the same syntax for format strings as the [strftime()]
 //! (http://man7.org/linux/man-pages/man3/strftime.3.html) function from the C
 //! standard library.
-
 #![doc(html_logo_url = "https://www.rust-lang.org/logos/rust-logo-128x128-blk-v2.png",
        html_favicon_url = "https://www.rust-lang.org/favicon.ico",
        html_root_url = "https://doc.rust-lang.org/time/")]
 #![allow(trivial_numeric_casts)]
+#![allow(unused_imports)]
 #![cfg_attr(test, deny(warnings))]
+#![cfg_attr(test, allow(deprecated))]
+#![cfg_attr(not(feature = "std"), no_std)]
 
 #[cfg(target_os = "redox")] extern crate syscall;
-#[cfg(unix)] extern crate libc;
+#[cfg(all(unix, feature = "std"))] extern crate libc;
 #[cfg(windows)] extern crate kernel32;
```

Comment:
I think these run the same problem with `libc`, right? This effectively means that `no_std` doesn't work on Windows?